Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -537,6 +536,71 @@ def create_event_command(demisto_args: dict):
     )
 
 
+def add_user_to_misp(demisto_args: dict = {}):
+    """Adding a new user to MISP.
+
+    Args:
+        demisto_args (dict): Demisto args
+    """
+    new_user = MISPUser()
+    new_user.email = demisto_args.get('email')
+    new_user.org_id = demisto_args.get('org_id')
+    new_user.role_id = demisto_args.get('role_id')
+    new_user.password = demisto_args.get('password')
+    response = PYMISP.add_user(new_user)
+    if 'errors' in response:
+        raise DemistoException(f'Failed to add user.\nError message: {response}')
+    else:
+        human_readable = f"## MISP add user\nNew user was added to MISP.\nEmail:{new_user.email}"
+        return CommandResults(readable_output=human_readable, raw_response=response, outputs=response)
```

Comment:
```suggestion
        return CommandResults(readable_output=human_readable, raw_response=response, outputs=response, outputs_prefix='MISP.Organization')
```
Please add the outputs_prefix param to the CommandResults in order that the output will match MISP.User.id